backend/apid/graphql/schema/asset.graphql

```diff
@@ -26,4 +26,10 @@ type Asset implements Node, Namespaced, HasMetadata {
   queries are joined by the "AND" operator.
   """
   filters: [String]
+
+  """
+  toJSON returns a REST API compatible representation of the resource. Handy for
+  sharing snippets that can then be imported with `sensuctl import`.
```

Comment:
What's `sensuctl import`? Do you mean `sensuctl create`?

Reply:
That's what I get for doing this quickly and copy and pasting it all over. 😅